<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>utilities</class>
 <widget class="QWidget" name="utilities">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>418</width>
    <height>484</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="utilitiesLayout">
   <item>
    <widget class="QLabel" name="probPlot">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>70</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>300</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
     <property name="buddy">
      <cstring>updateButton</cstring>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="goalLabel">
     <property name="text">
      <string>Goal:</string>
     </property>
     <property name="buddy">
      <cstring>goalList</cstring>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QComboBox" name="goalList">
     <item>
      <property name="text">
       <string>&lt;Select&gt;</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Detect</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Map</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Navigate</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Review status</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Inspect current site</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Explore</string>
      </property>
     </item>
    </widget>
   </item>
   <item>
    <widget class="QStackedWidget" name="probTables">
     <property name="locale">
      <locale language="English" country="UnitedStates"/>
     </property>
     <property name="currentIndex">
      <number>6</number>
     </property>
     <widget class="QWidget" name="select">
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="margin">
        <number>1</number>
       </property>
      </layout>
     </widget>
     <widget class="QWidget" name="detect">
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="margin">
        <number>1</number>
       </property>
       <item>
        <widget class="QTableView" name="detectTable"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="mapping">
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="margin">
        <number>1</number>
       </property>
       <item>
        <widget class="QTableView" name="mappingTable"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="navigate">
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QTableView" name="navigateTable"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="review">
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <widget class="QTableView" name="reviewTable"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="inspect">
      <layout class="QVBoxLayout" name="verticalLayout_6">
       <item>
        <widget class="QTableView" name="inspectTable"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="explore">
      <layout class="QVBoxLayout" name="verticalLayout_7">
       <item>
        <widget class="QTableView" name="exploreTable"/>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="updateLayout">
     <property name="spacing">
      <number>0</number>
     </property>
     <property name="margin">
      <number>1</number>
     </property>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="updateButton">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>32</height>
        </size>
       </property>
       <property name="text">
        <string>Update</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>goalList</tabstop>
  <tabstop>updateButton</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>updateButton</sender>
   <signal>clicked()</signal>
   <receiver>probPlot</receiver>
   <slot>update()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>342</x>
     <y>462</y>
    </hint>
    <hint type="destinationlabel">
     <x>144</x>
     <y>49</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>goalList</sender>
   <signal>activated(int)</signal>
   <receiver>probTables</receiver>
   <slot>setCurrentIndex(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>208</x>
     <y>121</y>
    </hint>
    <hint type="destinationlabel">
     <x>208</x>
     <y>285</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
